examples/word_of_day.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./word_of_day_template');
+const url = require('url');
+
+const home_page = 'http://www.thefreedictionary.com/_/archive.htm'; // Home section
+
+
+function ingest_article_profile(hatch, uri) {
+    return libingester.util.fetch_html(uri).then(($profile) => {
+        const base_uri = libingester.util.get_doc_base_uri($profile, uri);
+        const asset = new libingester.NewsArticle();
+
+        asset.set_canonical_uri(uri);
+
+        // use url library to pull out query string date object
+        const parts = url.parse(uri, true);
+        const modified_date = new Date(Date.parse(parts.query.d));
+
+        asset.set_last_modified_date(modified_date);
+        asset.set_section("word_of_day");
+
+        //Set title section
+        const date = parts.query.d;
+        const title = $profile('h1').first().text() + ": " + date;
+
+        // Pluck wordOfDay
+        const wordOfDay = $profile('table.widget', 'div#colleft').first().find('h3').children().text();
+        asset.set_title(wordOfDay);
+
+        // Pluck wordOfDayDef
+        const wordOfDayMixedString = $profile('td:contains("Definition:")').next().text();
+        const wordOfDayDef = wordOfDayMixedString.split(" ").slice(1).join(" ");
```

Comment:
This seems too messy of logic. Try to slice the string from first space onward with `indexOf()` as the begging.